src/app/project/comments/comments.component.ts

```diff
@@ -101,13 +101,13 @@ export class CommentsComponent implements OnInit, OnDestroy {
           if (data.commentPeriod) {
             // To fix the issue where the last page is empty.
             this.commentPeriod = data.commentPeriod;
-            console.log('Comment Period info', this.commentPeriod);
+            const engagementLabel = this.project.engagementLabel ? this.project.engagementLabel : 'Public Comment Period';
```

Comment:
just curious, || doesn't work here?

Reply:
Ahh, good point. That would have been less verbose. I think I had meant to switch this to the OR operator but never came back to it.